tests/test_client.py

```diff
@@ -248,10 +253,18 @@ def test_success_merge_profile_idempotent():
     }
 
     c = Courier(auth_token='123456789ABCDF')
-    r = c.merge_profile("1234", profile, idempotency_key="1234ABCD")
+    x = (datetime.now()+timedelta(days=7)).isoformat()
```

Comment:
lets call this variable something more readable like maybe `expiration_date` or something.

Reply:
also applies to other tests.